src/main/java/ch/uzh/ifi/hase/soprafs22/lobby/LobbyManager.java

```diff
@@ -94,8 +98,27 @@ public void clear() {
         lobbyMap.clear();
     }
 
+    @Override
+    public void modifyPlayer(String token, Long lobbyId, PlayerPutDTO playerPutDTO) {
+        ILobby lobby = lobbyMap.get(lobbyId);
+        String newName = playerPutDTO.getName();
+        Boolean ready = playerPutDTO.getReady();
+        if (newName != null) {
+            if (isEmpty(newName))
+                throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Username should not be empty.");
```

Comment:
ResponseStatusException in the LobbyManager? I'm not sure about that...

Reply:
Done